common/nodejs-utils/src/package_json.rs

```diff
@@ -1,16 +1,20 @@
 use crate::vrs::{Requirement, Version};
 use serde::Deserialize;
+use std::collections::HashMap;
 use std::fs::File;
 use std::io::BufReader;
 use std::path::Path;
 use thiserror::Error;
 
 #[derive(Deserialize, Debug)]
 pub struct PackageJson {
-    pub name: String,
+    pub name: Option<String>,
```

Comment:
@schneems this should help with #371 
cc: @joshwlewis @edmorley

Reply:
i did add a changelog entry but i can revert those if that's your recommendation.